shm/player.cpp

```diff
@@ -53,34 +41,17 @@ void Player::nextDay(){
     payCrew(ship_->getCrew());
 }
 
-void  Player::giveCargo(Alcohol* cargo) {
-    ship_->load(cargo);
-}
-
-void  Player::giveCargo(Fruit* cargo) {
-    ship_->load(cargo);
-}
-
-void  Player::giveCargo(Item* cargo) {
-    ship_->load(cargo);
-}
-
-void  Player::removeAlco(Alcohol* cargo) {
-    ship_->removeAlco(cargo);
-}
-
-void  Player::removeFruit(Fruit* cargo) {
-    ship_->removeFruit(cargo);
-}
-
-void  Player::removeItem(Item* cargo) {
-    ship_->removeItem(cargo);
+size_t Player::getAvailableSpace() { 
+    return availableSpace_ = ship_->countAvailableSpace();
```

Comment:
Do we need availableSpace_ in the player class. Can't we just return ship_->countAvailableSpace()?

Reply:
Yes, you're right. Maybe this field will be useful in the future, when the player will have more ships :)